src/js/Controllers/UIController.js

```diff
@@ -157,7 +157,6 @@ class UIController {
                 
                 if (rpc.params.functionID === 10 && app.isPerformingInteraction
                      && (rpc.params.cancelID === undefined || rpc.params.cancelID === app.interactionCancelId)) {
-                    this.listener.send(RpcFactory.VRPerformInteractionAbortedResponse(app.interactionId - 1))
```

Comment:
Probably going to need to remove the existing response as well.

Reply:
removed in https://github.com/smartdevicelink/generic_hmi/pull/176/commits/ce398d2e787eb86b7e2a59acdfaac9a562b40fce